### ISSUE TITLE
Husky is required to install since 1.3.1

### ISSUE THREAD

**Issue by vitoyucepi** [storeon/localstorage#48]
## Summary
Since version 1.3.1 `husky` is required to install `@storeon/localstorage` as a dependency.

But it's not defined as dependency. So I have to manually add it as a dependency to my project if I didn't use it.
## Logs
Yarn output:
```
error /project/path/here/node_modules/@storeon/localstorage: Command failed.
Exit code: 127
Command: husky install
Arguments: 
Directory: /project/path/here/node_modules/@storeon/localstorage
Output:
/bin/sh: line 1: husky: command not found
```
## My setup
| Package | Version |
| --- | --- |
| `node` | `12.21.0` |
| `npm` | `7.6.2` |
| `yarn` | `1.22.10` |
| `storeon` | `3.1.4` |
| `@storeon/localstorage` | `1.3.0` |
| `husky` | `N/A` |

**Comment by polemius** [comment]
@vitoyucepi Thank you for the issue. Sorry about that. Just release a new 1.4.0 version without husky. Please try the new version.

**Comment by vitoyucepi** [comment]
Maybe husky is ok, but postinstall script goes into npm package.
```json
{
  "name": "@storeon/localstorage",
  "version": "1.3.1",
  "description": "Module for storeon to persist data from state to local storage",
  "repository": "storeon/localstorage",
  "author": "Ivan Menshykov <ivan.menshykov@gmail.com>",
  "license": "MIT",
  "scripts": {
    "postinstall": "husky install"
  },
  "type": "module",
  "main": "index.cjs",
  "module": "index.js",
  "react-native": "index.js",
  "exports": {
    ".": {
      "require": "./index.cjs",
      "import": "./index.js"
    },
    "./package.json": "./package.json"
  }
}
```

**Comment by vitoyucepi** [comment]
Quote from docs https://typicode.github.io/husky/#/?id=yarn-v2
```
if your package is not private and you're publishing it on a registry like npmjs.com, you need to disable `postinstall` script using `pinst`. Otherwise, `postinstall` will run when someone installs your package and result in an error.
```

**Comment by vitoyucepi** [comment]
If you're ok with `simple-git-hooks` then issue is solved, because I've tested `1.4.0` and it works for me.

**Comment by polemius** [comment]
@vitoyucepi thank you letting me know. Yes, I am ok with `simple-git-hooks`.
